docs/user_guide/docker/docker.md

```diff
@@ -0,0 +1,76 @@
+
+## How to Upload Docker Images
+
+As an alternative to syncing content, Docker can images can be uploaded directly into Katello either by the UI or the CLI. These images can either be added to Repositories which have been synced from an external source such as Docker Hub or they can be uploaded into Repositories which have no external source. For the latter, you can create a Repository in Katello with no upstream name or registry URL. This is handy if you just want to create a repository from only Docker images you plan to upload.
+
+In order to upload a Docker image, you'll first need to export it from Docker. To do this, you'll use the Docker `save` command.
+
+```
+docker save centos:latest > centos_latest.tar
+```
+
+This tar file can now be directly uploaded into Katello.
+
+To upload an image in the UI, simply visit the Repository details page for the Repository. On the page, there's a section "Upload Docker Image." In this section, click the file field and select your image(s) and click upload. You should get a response indicating whether the file upload has succeeded or failed.
+
+To upload images from the CLI, you must have [hammer and hammer-cli-katello installed and configured](../../cli/index.html). After hammer is properly setup and configured, the follow command can be used to upload your image:
+
+```
+$ hammer repository upload-content --path centos_latest.tar --id 6
+```
+
+## How to Publish and Promote Docker Content
+
+Docker content can be published and promoted via Content Views much like yum or puppet content.
+
+![Adding Docker Repositories to a Content View](docker_cv.png)
+
+After creating a Content View, visit the Docker Content tab. Here you can select any Docker repositories you want to add to your Content View. After you've added Docker Repositories to your view, you may proceed as normal. Visit [the Content View user guide for more information](../content_views/content_views.html).
+
+## How to View and Pull Docker Content
+
+To view Docker content contained with Katello, visit the Docker Tags page. This can be accessed under the Content menu at the top of any page.
+
+![Docker Tags Page](docker_tags.png)
+
+On the Docker Tags page, you can see a list of Docker Tags grouped by Repository in Katello. This shows you Tags grouped across Content Views and Lifecycle Environments. Suppose I wanted to pull the latest Tag from my redis repository, I would click the latest row for my redis repository.
+
+![Docker Tags Page](docker_tags.png)
+
+I can see here that my redis Repository has been added to a published Content View called redisv. If I want to use the tag from that Content View, I would just copy the Published At URL and then on my docker server I would run:
+
+```
+$ docker pull localhost:5000/default_organization-library-redisv-Tester-redis:latest
+Pulling repository localhost:5000/default_organization-library-redisv-Tester-redis...
+```
+
+## How to Provision Docker Content
+
+TBD.
```

Comment:
Am waiting to see what foreman posts up in terms of documentation and tie this into their docs (with links, etc)